lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3568,6 +3589,24 @@ namespace Js
         return (flags != ImplicitCall_None) || arrayIteratorPrototypeNext != scriptContext->GetLibrary()->GetArrayIteratorPrototypeBuiltinNextFunction();
     }
 
+    bool JavascriptLibrary::InitializeBigIntConstructor(DynamicObject* bigIntConstructor, DeferredTypeHandlerBase * typeHandler, DeferredInitializeMode mode)
+    {
+        typeHandler->Convert(bigIntConstructor, mode, 3);
```

Comment:
Can you add a comment describing what the magic number 3 does (or use a named constant here?)

Reply:
will do